unified-scheduler-pool/src/lib.rs

```diff
@@ -1280,24 +1329,97 @@ impl<S: SpawnableScheduler<TH>, TH: TaskHandler> ThreadManager<S, TH> {
     }
 
     #[must_use]
-    fn accumulate_result_with_timings(
+    fn abort_or_accumulate_result_with_timings(
```

Comment:
request for doc comment:
```rust
    /// Returns `true` if the caller should abort.
```

Reply:
done: 4327ccd4e6200d93b87b361aa3f582a630ba2ba3